tests/bsim/bluetooth/host/misc/sample_test/README.rst

```diff
@@ -0,0 +1,34 @@
+.. _bluetooth_bsim_test_sample:
+
+Bluetooth: Example BabbleSim test
+#################################
+
+Abstract
+********
+
+This test's purpose is to serve as template for implementing a new BabbleSim Bluetooth test.
+
+BabbleSim_ is :ref:`integrated in zephyr <bsim>` and used for testing the Bluetooth stack.
+The tests are implemented in ``tests/bsim/bluetooth``.
+They can unfortunately only be run on Linux (for now).
```

Comment:
```suggestion
They can only be run on Linux.
```
This is not changing any time soon, so let's not set the wrong expectations

Reply:
what's stopping us from porting it to a *bsd?